internal/controller/controllers/clusterdeployments_controller.go

```diff
@@ -444,11 +444,7 @@ func (r *ClusterDeploymentsReconciler) updateInstallConfigOverrides(ctx context.
 }
 
 func (r *ClusterDeploymentsReconciler) getOCPVersion(cluster *hivev1.ClusterDeployment) string {
```

Comment:
do we still need this function? maybe should just be a const?

Reply:
I think that it still be used by @danielerez if not then i will remove it.

Reply:
Should be safe the remove (using the release version from OpenshiftVersion entity instead).

Reply:
Removed and replaced with a const.

Reply:
I think it can stay and we can add logic here once we start supporting 4.8.x and 4.9...